cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-simple-cli-guide.adoc

```diff
@@ -0,0 +1,45 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-getting-started-simple-cli-guide"]
+= Tutorial: Simple CLI guide
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-getting-started-simple-cli-guide
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2023-11-16
+
+This page outlines the minimum list of commands to deploy a ROSA cluster using the command line interface (CLI). 
+
+[NOTE]
+====
+While this simple deployment works well for a workshop setting, clusters used in production should be deployed with a more detailed method.
+====
+
+== Prerequisites
```

Comment:
Should "Prerequisites" be formatted in this way? 
![image](https://github.com/openshift/openshift-docs/assets/122639474/87700ac0-2b20-4533-9ef7-4e1801695b98)